### DISCUSSION TITLE
Segmentation fault (core dumped)

### DISCUSSION THREAD

**Discussion by huisky2000** [OpenRadioss/OpenRadioss#493] [Users]
Hi,
Try to run several examples, always end up with the same error. why?
I'm running on windows subsystem for linux (WSL). Anything wrong?

Run:  ../../exec/starter_linux64_gf -i fsi_drop_container_0000.rad

WARNING ID : 100214
** WARNING IN INPUT OPTIONS
DESCRIPTION :
-- BLOCK: /IOFLAG
-- LINE: 1                   0         0         0         0         0
unsupported field exits
SOLUTION :
    Please check input format and its consistency with Invers value from /BEGIN option.

WARNING ID : 100213
** WARNING IN INPUT OPTIONS
DESCRIPTION :
-- BLOCK: /INIVOL/86/10003507
-- LINE: 998         3         1         0                 1.0
unsupported field exits at the end of line
SOLUTION :
    Please check input format and its consistency with Invers value from /BEGIN option.

WARNING ID : 100213
** WARNING IN INPUT OPTIONS
DESCRIPTION :
-- BLOCK: /INIVOL/86/10003507
-- LINE: 9999         2         0         0                 1.0
unsupported field exits at the end of line
SOLUTION :
    Please check input format and its consistency with Invers value from /BEGIN option.

 .. UNITS SYSTEM
 .. CONTROL VARIABLES
 .. STARTER RUNNING ON    4 THREADS

WARNING ID :   1032
** WARNING STACKSIZE LIMIT IS LOWER THAN 512M

 .. FUNCTIONS & TABLES
 .. MATERIALS

WARNING ID :     98
** WARNING IN MATERIAL LAW51
 .. NODES
 .. SKEW SYSTEMS
 .. PROPERTIES
 .. 3D SOLID ELEMENTS
 .. 3D SHELL ELEMENTS
 .. 3D TRIANGULAR SHELL ELEMENTS
 .. SUBSETS
 .. ELEMENT GROUPS
 .. PART GROUPS
 .. SURFACES
 .. NODE GROUP

WARNING ID :    690
** WARNING IN NODE GROUP DEFINITION
 .. BOUNDARY CONDITIONS
 .. ALE/THE TABLES
 .. INITIAL VELOCITIES
 .. INITIAL VOLUME FRACTION
 .. DOMAIN DECOMPOSITION
 .. ELEMENT GROUPS
 .. INTERFACES
 .. INTERFACE BUFFER INITIALIZATION
 .. RIGID WALLS
 .. RETURNS TO DOMAIN DECOMPOSITION FOR OPTIMIZATION
 .. DOMAIN DECOMPOSITION
 .. ELEMENT GROUPS
 .. INTERFACES
 .. INTERFACE BUFFER INITIALIZATION
 .. RIGID WALLS
 .. ELEMENT BUFFER INITIALIZATION
Segmentation fault (core dumped)

**Comment by PaulAltair** [comment]
looks like you are hitting memory limit, can you set `stacksize` to unlimited and try again?

**Comment by MarianBulla** [comment]
Can you check your wsl version, please ?
Just open a windows command shell and type "wsl -l -v"
It should show you VERSION   2.
If it is still "1", then you have to update it, using a command shell with andmin rights.

**Reply by huisky2000** [reply]
yes, u r right! It is now working after upgrade wsl from 1 to 2.
Thanks!
